apis/voyager/v1beta1/validator.go

```diff
@@ -274,6 +296,31 @@ func checkOptionalPort(port intstr.IntOrString) (int, error) {
 	return 0, fmt.Errorf("invalid data type %v for port %s", port.Type, port)
 }
 
+func checkOptionalAddress(address string) (string, error) {
+	if address != "" && net.ParseIP(address) == nil {
+		return "", fmt.Errorf("could not parse IPv4 or IPv6 address")
+	} else if address == "" {
+		return "*", nil
+	}
+
+	return address, nil
+}
+
+func checkExclusiveWildcard(address string, port int, defined map[string]*address) error {
+	var wildcard = fmt.Sprintf("*:%d", port)
+	if address == "*" {
+		if _, ok := defined[wildcard]; !ok && len(defined) > 0 {
+			return fmt.Errorf("cannot define wildcard on same port as specific bind address")
+		}
+	} else {
+		if _, ok := defined[wildcard]; ok {
+			return fmt.Errorf("cannot define specific bind address on same port as wildcard")
```

Comment:
`Can't use specific bind address for port %d, since ones or more rules use wildcard bind address.`